src/o365/spfx/commands/project/project-upgrade.ts

```diff
@@ -374,6 +379,20 @@ class SpfxProjectUpgradeCommand extends BaseProjectCommand {
     cb();
   }
 
+  private writeReportTourFolder(findingsToReport: any, cmd: CommandInstance, options: Options): void {
+    const toursFolder: string = path.resolve(`${this.projectRootPath}/.tours`);
+
+    if (!fs.existsSync(toursFolder)) {
+      fs.mkdirSync(toursFolder, { recursive: false });
+    }
+
+    // Override reports folder
+    options.outputFile = `${this.projectRootPath}/.tours/upgrade.tour`;
```

Comment:
We should avoid constructing paths manually and use `path.join` instead which is more reliable across the different OSes